db/schema.rb

```diff
@@ -19,8 +19,8 @@
     t.string "email", default: "", null: false
     t.string "encrypted_password", default: "", null: false
     t.string "reset_password_token"
-    t.datetime "reset_password_sent_at"
-    t.datetime "remember_created_at"
+    t.datetime "reset_password_sent_at", precision: 6
```

Comment:
Where is this coming from?

Reply:
No idea, just running the migrations does that

Reply:
Curious, seems to have happened on all date fields... anything underlying changed?

Reply:
Not that I know of, I'll manually remove them cause it's a bit odd

Reply:
Maybe Postgres version is different (? dunno if that'd impact it)